src/Hamiltonians/HubbardMom1D.jl

```diff
@@ -161,9 +161,9 @@ end
     get_offdiagonal(ham, add, chosen, num_singly_doubly_occupied_sites(add)...)
 end
 
-@inline function get_offdiagonal(
-    ham::HubbardMom1D{<:Any,M,A}, add, chosen, singlies, doublies
-) where {M,A}
+"Internal function used in `get_offdiagonal()`, `-q` is the change in momentum needed for `G2Corrlator`."
```

Comment:
```suggestion
"""
    generate_new_add(add, chosen, singlies, doublies)
Internal function used in [`get_offdiagonal`](@ref). Returns the new address, the onproduct,
and the change in momentum, which is needed for [`G2Correlator`](@ref). 
"""
```

Reply:
I know this is just an internal function, but I would prefer to see a descriptive name and accurate docstring nevertheless. I see that this is used for both the G2 correlator and for `HubbardMom1D`. What about `momentum_transfer_excitation`, which I think is what it creates? 
```suggestion
"""
    momentum_transfer_excitation(add, chosen, singlies, doublies)
Internal function used in [`get_offdiagonal`](@ref) for [`HubbardMom1D`](@ref) 
and [`G2Correlator`](@ref). Returns the new address, the onproduct, 
and the change in momentum. 
"""
```